pydarn/io/borealis/borealis.py

```diff
@@ -189,6 +188,14 @@ def arrays(self):
         """
         return self._reader.arrays
 
+    @property
+    def borealis_version(self):
+        """
+        The Borealis version that created this data. May impact the
+        fields included in the file.
+        """
+        return self._reader._borealis_version
```

Comment:
This version refers to the version of the Borealis software package. I'll update the doc for that also, but I think it might not be as clear if it's just version, since it could be version of multiple things. It kinds of reads as if there are multiple versions of the reader.

Reply:
maybe `software_version` then? Again the package and class name is Borealis already so people may already know they are working with Borealis code related things.

Reply:
agreed replace with software version, will do.